src/current/v23.1/advanced-changefeed-configuration.md

```diff
@@ -33,6 +33,42 @@ kvadmission.rangefeed_catchup_scan_elastic_control.enabled
 
 For a more technical explanation of elastic CPU, refer to the [Rubbing control theory on the Go scheduler](https://www.cockroachlabs.com/blog/rubbing-control-theory/) blog post.
 
+### Latency in changefeeds
+
+When you are running large workloads, changefeeds can encounter or cause latency in a cluster in the following ways:
+
+- Changefeeds can have an impact on SQL latency in the cluster generally.
+- Changefeeds can encounter latency in **events** emitting. This latency is the total time CockroachDB takes to:
+    - Commit writes to the database.
+    - Encode [changefeed messages]({% link {{ page.version.version }}/changefeed-messages.md %}).
+    - Deliver the message to the [sink]({% link {{ page.version.version }}/changefeed-sinks.md %}).
+
+The following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) reduce bursts of [rangefeed]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds) work so that updates are paced steadily over time.
+
+{{site.data.alerts.callout_danger}}
+We do **not** recommend adjusting these settings unless you are running a large workload, or are working with the Cockroach Labs [support team]({{ link_prefix }}support-resources.html).
+{{site.data.alerts.end}}
+
+#### `kv.closed_timestamp_target_duration`
+
+**Default:** `3s`
+
+This setting controls the frequency of checkpoints for each [range]({% link {{ page.version.version }}/architecture/overview.md %}#range). A changefeed aggregates these checkpoints across all ranges, and once the timestamp on all the ranges advances, the changefeed can then [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). As a result, the higher the value of this setting the longer it can take for a changefeed to checkpoint. It is important to note that a changefeed at default configuration does not checkpoint more often than once every 30 seconds.
+
+In clusters running large-scale workloads, increasing this setting can help to lower the potential impact of changefeeds on SQL latency. That is, an increase in the setting could lower the load on the cluster. This is important for workloads with tables in the TB range of data. However, for most workloads, we recommend leaving this setting at the default of `3s`.
+
+{{site.data.alerts.callout_danger}}
```

Comment:
should this warning be placed after both cluster settings?

Reply:
This one warning I kept to the upper setting because the lower one we really don't anticipate customers changing. I have altered part of the language in that setting's description to ensure it's a little clearer that the default is actually good here.